applications/ContactStructuralMechanicsApplication/custom_includes/point_item.h

```diff
@@ -142,9 +142,19 @@ class PointItem: public Point<3>
      * @return Coordinates: The coordinates of the item
      */
 
-    void UpdatePoint()
-    {
-        this->Coordinates() = mpOriginCond->GetGeometry().Center().Coordinates();
+    void UpdatePoint(const double& DeltaTime)
+    {        
+        Point<3> center;
+        if (mpOriginCond->GetGeometry()[0].SolutionStepsDataHas(VELOCITY_X) == true)
+        {
+            center = ContactUtilities::GetHalfJumpCenter(mpOriginCond->GetGeometry(), DeltaTime); // NOTE: Center in half delta time
+        }
+        else
+        {
+            center = mpOriginCond->GetGeometry().Center(); // NOTE: Real center
+        }
+        
+        this->Coordinates() = center.Coordinates();
```

Comment:
Very dangerous. This will easily break parallelism

Reply:
what do you recommend?

Reply:
There is a function AddExplicitCoontribution or smug similar for this type of things....

Reply:
This commentary is about the addition in the nodal value?, or is related to

~~~c
this->Coordinates() = center.Coordinates();
~~~

Reply:
Is anything wrong with this code?

Reply:
I change the coordinates of point that I created myself, nothing already existing